README.md

```diff
@@ -78,8 +78,11 @@ be created in a private repository created from this template repository.
 
     The output will be used in the next step.
 
+1. Assing a `Application.ReadWrite.OwnedBy` role to the Service Principal you've just created. 
```

Comment:
Just a drive-by as I had to go through this

Reply:
this is covered in the `make ci-auth` in #95 now

Reply:
Cool, do you want me to remove this?